firebase-installations/src/androidTest/java/com/google/firebase/installations/FirebaseInstallationsInstrumentedTest.java

```diff
@@ -264,11 +266,12 @@ public void testGetId_PersistedFidError_BackendOk() throws InterruptedException
     // Expect exception
     try {
       Tasks.await(firebaseInstallations.getId());
-      fail();
+      fail("Could not update local storage.");
     } catch (ExecutionException expected) {
       Throwable cause = expected.getCause();
```

Comment:
now that this local cause variable is only being used in one place, I'd just inline it